Sources/TokamakStaticHTML/Shapes/_ShapeView.swift

```diff
@@ -37,9 +37,9 @@ extension _ShapeView: ViewDeferredToRenderer {
     if let shapeAttributes = shape as? ShapeAttributes {
       return AnyView(HTML("div", shapeAttributes.attributes(style)) { path })
     } else if let color = style as? Color {
-      return AnyView(HTML("div", ["style": "fill: \(color);"]) { path })
+      return AnyView(HTML("div", ["style": "fill: \(color.cssValue(environment));"]) { path })
     } else if let foregroundColor = foregroundColor {
-      return AnyView(HTML("div", ["style": "fill: \(foregroundColor);"]) { path })
+      return AnyView(HTML("div", ["style": "fill: \(foregroundColor.cssValue(environment));"]) { path })
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakStaticHTML/Shapes/_ShapeView.swift;
  Line: 42;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 104 characters (`line_length`)